src/storage/v2/storage.hpp

```diff
@@ -305,6 +309,14 @@ class Storage {
     return PropertyId::FromUint(name_id_mapper_->NameToId(name));
   }
 
+  std::optional<PropertyId> NameToPropertyIfExists(const std::string_view name) const {
```

Comment:
I would rename it to `GetNameToProperty`, makes more sense to me

Reply:
I mean it's not a getter, because you are doing a transformation, name -> property